lib/ProfilePicture/ProfilePicture.js

```diff
@@ -0,0 +1,47 @@
+import { useIntl } from 'react-intl';
+import PropTypes from 'prop-types';
+import { Img } from 'react-image';
+
+import { Loading } from '@folio/stripes-components';
+
+import { isAValidURL } from './utils/util';
+import useProfilePicture from './utils';
+import profilePicThumbnail from './icons/profilePicThumbnail.png';
```

Comment:
Would `placeholderThumbnail` be a better name for this? That might make it easier to distinguish between the two conditions (user data is present vs user data is missing).